packages/sources/src/addSourceWizard/hardcodedSchemas.js

```diff
@@ -204,21 +204,26 @@ export default {
                             label: 'Storage account name'
                         }]
                     }, {
-                        title: 'Configure service principal',
-                        name: 'service-principal',
+                        title: 'Enter Azure subscription ID',
+                        name: 'azure-sub-id',
+                        nextStep: 'configure-roles',
+                      fields: [{
+                        name: 'azure-sub-id-description',
+                        component: 'description',
+                        Content: CMAzure.SubscriptionID,
+                      }, {
+                        name: 'credentials.subscription_id',
+                        component: componentTypes.TEXT_FIELD,
+                        label: 'Subscription ID'
```

Comment:
Please move this label to 'credentials.subscription_id' object one level up - those objects are used for getting info in the summary.

Reply:
I thought I did :) in line 129.